src/redux/api/api.ts

```diff
@@ -0,0 +1,31 @@
+import 'firebase/database';
+import {database} from "../../firebase/firebaseConfig"
+import {get, ref, set} from "firebase/database";
+
+// Get Next FLashCards
+export const getNextFlashcard = async () => {
+    try {
+        let word
+        const snapshot = await get(ref(database, 'words'))
+        word = Object.values(snapshot.val())[0]
+        return word
```

Comment:
why do you need `word` variable at all here? You can just return it right away
```suggestion
        const snapshot = await get(ref(database, 'words'))
        return Object.values(snapshot.val())[0]
```